portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -113,10 +143,7 @@ private String convertToJsonUsingGson(List toConvert) {
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    UserService userService = UserServiceFactory.getUserService();
-    String email = userService.getCurrentUser().getEmail();
-    System.out.println(email);
-
+    String imageUrl = getUploadedFileUrl(request, "image").orElse("");
     String text = request.getParameter(COMMENT_INPUT);
     long timestamp = System.currentTimeMillis();
     String name = (String) request.getParameter(COMMENT_NAME);
```

Comment:
nit: it is cleaner if these are declared in the order they are used.

Reply:
Done!